CalibMatCorrection.m

```diff
@@ -49,7 +49,7 @@
         else
             scriptOptions.firstTime=false;
         end
-        if (scriptOptions.firstTime)
```

Comment:
i have this check because if someone does not use the identity name but the calibration matrix is the identity it should still enter it can not depend only in the trust of the user of putting identity in the calibmatfile name. If you see firsttime is set just before this if